src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/FunctionalTests.cs

```diff
@@ -35,22 +35,64 @@ public static IEnumerable<object[]> GetDataForHl7v2()
+                new[] { @"ADT_A29", @"ADT_A29.hl7", @"ADT-A29-expected.json" },
+                new[] { @"ADT_A29", @"ADT_A29-2.hl7", @"ADT-A29-expected-2.json" },
                 new[] { @"ADT_A31", @"ADT_A31.hl7", @"ADT-A31-expected.json" },
-                new[] { @"ADT_A31", @"ADT_A31-2.hl7", @"ADT-A31-expected-2.json" },     
+                new[] { @"ADT_A31", @"ADT_A31-2.hl7", @"ADT-A31-expected-2.json" },
                 new[] { @"ADT_A47", @"ADT_A47.hl7", @"ADT-A47-expected.json" },
                 new[] { @"ADT_A47", @"ADT_A47-2.hl7", @"ADT-A47-expected-2.json" },
                 new[] { @"ADT_A60", @"ADT_A60.hl7", @"ADT-A60-expected.json" },
                 new[] { @"ADT_A60", @"ADT_A60-2.hl7", @"ADT-A60-expected-2.json" },
 
+                new[] { @"SIU_S12", @"SIU-S12-01.hl7", @"SIU-S12-01-expected.json" },
+                new[] { @"SIU_S12", @"SIU-S12-02.hl7", @"SIU-S12-02-expected.json" },
+                new[] { @"SIU_S13", @"SIU-S13-01.hl7", @"SIU-S13-01-expected.json" },
+                new[] { @"SIU_S13", @"SIU-S13-02.hl7", @"SIU-S13-02-expected.json" },
+                new[] { @"SIU_S14", @"SIU-S14-01.hl7", @"SIU-S14-01-expected.json" },
+                new[] { @"SIU_S14", @"SIU-S14-02.hl7", @"SIU-S14-02-expected.json" },
+                new[] { @"SIU_S15", @"SIU-S15-01.hl7", @"SIU-S15-01-expected.json" },
+                new[] { @"SIU_S15", @"SIU-S15-02.hl7", @"SIU-S15-02-expected.json" },
+                new[] { @"SIU_S16", @"SIU-S16-01.hl7", @"SIU-S16-01-expected.json" },
+                new[] { @"SIU_S16", @"SIU-S16-02.hl7", @"SIU-S16-02-expected.json" },
+                new[] { @"SIU_S17", @"SIU-S17-01.hl7", @"SIU-S17-01-expected.json" },
+                new[] { @"SIU_S17", @"SIU-S17-02.hl7", @"SIU-S17-02-expected.json" },
+                new[] { @"SIU_S26", @"SIU-S26-01.hl7", @"SIU-S26-01-expected.json" },
+                new[] { @"SIU_S26", @"SIU-S26-02.hl7", @"SIU-S26-02-expected.json" },
+
+                new[] { "ORU_R01", "ORU-R01_1.hl7",  @"ORU-R01_1-expected.json"},
+
+                new[] { @"ORM_O01", @"ORM_O01_01.hl7", @"ORM_O01_01-expected.json" },
+                new[] { @"ORM_O01", @"ORM_O01_02.hl7", @"ORM_O01_02-expected.json" },
+                new[] { @"ORM_O01", @"ORM_O01_03.hl7", @"ORM_O01_03-expected.json" },
+
+                new[] { "MDM_T01", "MDM-T01-01.hl7",  @"MDM-T01-01-expected.json"},
+                new[] { "MDM_T01", "MDM-T01-02.hl7",  @"MDM-T01-02-expected.json"},
+                new[] { "MDM_T02", "MDM-T02-01.hl7",  @"MDM-T02-01-expected.json"},
+                new[] { "MDM_T02", "MDM-T02-02.hl7",  @"MDM-T02-02-expected.json"},
+                new[] { "MDM_T02", "MDM-T02-03.hl7",  @"MDM-T02-03-expected.json"},
+
+                new[] { "OML_O21", "OML_O21_01.hl7",  @"OML_O21-01-expected.json"},
+                new[] { "OML_O21", "OML_O21_02.hl7",  @"OML_O21-02-expected.json"},
+
```

Comment:
Nit: Can we unify the file names of the sample data and expected results? Currently some file names are using `-` to combine words while some others are using `_`. Some of them are named `template_{number}-expected.json` while some others are named `template-expected-{number}`.

Reply:
Sure , We will unify the file names and will push the changes for it

Reply:
@BoyaWu10 : We have unified the name of the sample data and expected results and pushed the changes for it

Reply:
Thanks! This looks much clearer.